insights/core/__init__.py

```diff
@@ -284,6 +285,36 @@ def get(self, item, default=None):
         return self.data.get(item, default)
 
 
+def get_xmlns(self, root):
+        return root.tag.strip("{").split("}")[0] if all(c in root.tag for c in ["{", "}"]) else ""
+
+
+class XMLParser(Parser):
```

Comment:
We can let the LegacyItemAccess mixin here on the base class:
` class XMLParser(Parser, LegacyItemAccess):`